games-engines/openmw/openmw-9999.ebuild

```diff
@@ -46,6 +48,9 @@ RDEPEND="
 		dev-qt/qtopengl:5
 		dev-qt/qtwidgets:5
 	)
+	$(lua_gen_cond_dep '
+		dev-lua/sol2[${LUA_USEDEP}]
+	')
```

Comment:
Given last comment, made me check here again. Being a header package, this doesn't need RDEPEND, and likely doesn't need `${LUA_USEDEP}` either if make this test-only.

OpenMW itself is looking into which lua to use and depend on, some single-target c++ headers don't mean much in this.